### ISSUE TITLE
Doesn't work for thread previews on PHPBB boards like forums.anandtech.com

### ISSUE THREAD

**Issue by jevansturner** [dmillz/Comic-Text#4]
Installed your Chrome plug-in and immediately enabled for *. Doesn't work for alt text specified by a TD entry. Many PHPBB messageboards show pop-up alt text to preview the contents of a thread before you click on it. It's especially useful when a thread has a non-descriptive title and you're just trying to get an idea what the thread is about. However, the pop-up text still disappears too quickly.

[edit]
The pop-up text isn't stored as an "alt" attribute. It's stored as a "title" attribute of a TD tag.
